consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.consensus.ratis;
+
+import org.apache.iotdb.consensus.IConsensus;
+import org.apache.iotdb.consensus.common.ConsensusGroupId;
+import org.apache.iotdb.consensus.common.Peer;
+import org.apache.iotdb.consensus.common.request.IConsensusRequest;
+import org.apache.iotdb.consensus.common.response.ConsensusGenericResponse;
+import org.apache.iotdb.consensus.common.response.ConsensusReadResponse;
+import org.apache.iotdb.consensus.common.response.ConsensusWriteResponse;
+
+import java.util.List;
+
+public class RatisConsensus implements IConsensus {
+
+  @Override
+  public void start() {}
+
+  @Override
+  public void stop() {}
+
+  @Override
+  public ConsensusWriteResponse Write(
+      ConsensusGroupId groupId, IConsensusRequest IConsensusRequest) {
+    return null;
```

Comment:
The write module and partition module call the interface of the consensus layer earlier, and if null is returned here, the code logic of these two modules may be affected. I think it would be better to just return a "success" response here. Or should we use the StandAloneConsensus protocol first?

Reply:
Yeah, `RatisConsensus` is still under development, which is expected to be completed within the next 2 weeks by me and @SzyWilliam. You should use `IConsensus consensusImpl = new StandAloneConsensus(id -> new EmptyStateMachine());` to implement your logic at present.

In addition, the interface of the query has not been completely determined, you are welcome to put forward more detailed requirements on the interface structure of reading and writing according to your own usage scenarios.

Reply:
`Null` is no longer returned